src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Hosting
+{
+    public static class WebAssemblyHostBuilderExtensions
+    {
+        /// <summary>
+        /// Adds a delegate for configuring the provided <see cref="ILoggingBuilder"/>. This may be called multiple times.
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="WebAssemblyHostBuilder" /> to configure.</param>
+        /// <param name="configureLogging">The delegate that configures the <see cref="ILoggingBuilder"/>.</param>
+        /// <returns>The same instance of the <see cref="WebAssemblyHostBuilder"/> for chaining.</returns>
+        public static WebAssemblyHostBuilder ConfigureLogging(this WebAssemblyHostBuilder hostBuilder, Action<ILoggingBuilder> configureLogging)
+        {
+            return hostBuilder.ConfigureServices((collection) => collection.AddLogging(builder => configureLogging(builder)));
```

Comment:
Couldn't this be written without introducing a new extension method - hostBuilder.Services.AddLogging(...)`?

Reply:
I would say this is nice in the sense that it keeps consistency with the ASP.NET Core pattern about it, but I don't feel super strongly about it. That said, it's a single method so I think we should not be concerned. Do you have any other concerns in mind?

Reply:
If possible, @pranavkm's suggestion (`hostBuilder.Services.AddLogging`) keeps us closer aligned to the simplified startup model that @rynowak was working us towards.

Reply:
Yeah, to clarify:

`hostBuilder.ConfigureLogging` -> good
`hostBuilder.ConfigureServices` -> meh